### DISCUSSION TITLE
Multiple worlds for isolated simulations

### DISCUSSION THREAD

**Discussion by tower120** [bevyengine/bevy#2237] [Q&A]
I want to make simulation which does not directly affect main game world.
It would be nice (from design perspective) for me to have separate world, systems, etc. for that simulation.

As example think of some sort of game-in-game experience where you can play some old arcade game in main game. That arcade game logic, would love to live in a separate world/"everything."

**Comment by mockersf** [comment]
there is a RFC ongoing to propose subworlds: https://github.com/bevyengine/rfcs/pull/16. Could you take a look how that would work for you?

**Reply by tower120** [reply]
Thanks. I shared my thoughts there.
